src/Data/List/Relation/Binary/Permutation/Setoid/Properties.agda

```diff
@@ -384,15 +384,18 @@ split v as bs p = helper as bs p (<-wellFounded (steps p))
   helper (a ∷ [])     bs (refl eq)    _ = [ a ]      , bs , eq
   helper (a ∷ b ∷ as) bs (refl eq)    _ = a ∷ b ∷ as , bs , eq
   helper []           bs (prep v≈x _) _ = [] , _ , v≈x ∷ ≋-refl
-  helper (a ∷ as)     bs (prep eq as↭xs) (acc rec) with helper as bs as↭xs (rec _ ≤-refl)
-  ... | (ps , qs , eq₂) = a ∷ ps , qs , eq ∷ eq₂
+  helper (a ∷ as)     bs (prep eq as↭xs) (acc rec)
+    with ps , qs , eq₂ ← helper as bs as↭xs (rec (n<1+n _))
```

Comment:
Could we avoid rewriting the `with`s in this PR please? If you would like to change the library style-guide then best to open an issue for it :+1: